src/_locales/en/messages.json

```diff
@@ -123,6 +123,14 @@
     "message": "Show notifications.",
     "description": "Show notifications before an event starts."
   },
+  "show_add_to_google_calendar_in_context_menu": {
+    "message": "Show 'Add to Google Calendar' in context menu",
```

Comment:
Let’s simplify this to:
"Add to Google Calendar from menu"

- Not everyone knows what a _context_ menu is
- Quotes are often mis-handled in translation.